packages/x-charts/src/ChartsTooltip/ChartsTooltipRoot.tsx

```diff
@@ -0,0 +1,14 @@
+import { styled } from '@mui/material/styles';
+import Popper from '@mui/material/Popper';
+
+/**
+ * The root component of the tooltip.
+ * @ignore - internal component.
+ */
+export const ChartsTooltipRoot = styled(Popper, {
+  name: 'MuiChartsTooltip',
+  slot: 'Root',
+})(({ theme }) => ({
+  pointerEvents: 'none',
+  zIndex: theme.zIndex.modal,
+}));
```

Comment:
I think duplicating this component is needed. 

Because this code allows to use the MUI theme with `styleOverrides`.

For now the `MuiChartsTooltip` is identified as the main tooltip. By sharing this component the following customization will impact both the main tooltip and the sliders

```js
MuiChartsTooltip: {
  styleOverrides: {
    root: { backgroundColor: 'red' }
  }
}
```

The issue is that `MuiChartsTooltip` naming that is also used in `MuiChartsTooltip.table`, `MuiChartsTooltip.row`, ...

I'm in favor of duplicating to keep `MuiChartsTooltip` as a reference to the main tooltip. And in v8 have a more precise naming to distinguish the tooltip from its content

Reply:
Makes sense 👍